<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamInfo</class>
 <widget class="QWidget" name="TeamInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
    <pointsize>14</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>48</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Match Info</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter match info here.</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="a">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Team Number:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="teamNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Match Number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="matchNumber">
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Event Code (e.g. OHMV)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="eventCode">
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>20</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
